packages/mdc-dom/externs.d.ts

```diff
@@ -24,3 +24,7 @@
 declare interface Element {
   msMatchesSelector?: (selector: string) => boolean;
 }
+
+declare interface Window {
```

Comment:
Does moving this from `typings` require more things to rely on `dom`? Are we sure we have the dependency everywhere it's needed? Does this now need to be included explicitly?

Reply:
Good question. In our repo, TypeScript doesn't seem to care where this file is. Some wrapper frameworks might run into problems though, and would need to add a dep on `mdc-dom`. Would you rather put this in `mdc-base`? (I actually considered doing that, but `mdc-dom` seemed more semantically correct. But maybe convenience is more important than semantics.)

@moog16 @mmalerba Is it OK that these types have the same names as existing types in `lib.dom.d.ts`? Does TypeScript somehow merge the two type definitions together or something? Or should we create a new interface with a different name that `extends Window`?

Reply:
Exactly, TS merges interfaces: https://www.typescriptlang.org/docs/handbook/declaration-merging.html

Reply:
Done. Moved from `mdc-dom` to `mdc-base`.